internal/storage/store.go

```diff
@@ -1,22 +1,26 @@
 package storage
 
 import (
+	"time"
+
 	"github.com/fabric8-services/fabric8-jenkins-proxy/internal/configuration"
 	"github.com/jinzhu/gorm"
-	"time"
 
 	"context"
 	"fmt"
+	// Importing postgres driver to connect to the database
+	// The underscore import is used for the side-effect of a package.
 	_ "github.com/lib/pq"
 	log "github.com/sirupsen/logrus"
 )
 
 var storeLogger = log.WithFields(log.Fields{"component": "store"})
 
+// Store includes all methods required to interact with the database
 type Store interface {
 	CreateRequest(r *Request) error
 	GetRequests(ns string) (result []Request, err error)
-	IncRequestRetry(r *Request) (errs []error)
+	IncreaseRequestRetry(r *Request) (errs []error)
```

Comment:
I don't think that "Increase" == "Inc"
Inc stand for increment which is different from increasing something. So either 'Inc' which might be common enough, or 'Incremement'

Reply:
I would change this to IncrementRequestRetry()